### DISCUSSION TITLE
Updating to latest supabase version.

### DISCUSSION THREAD

**Discussion by RafaelCENG** [supabase/supabase#18787] [Questions]
This question will sound a bit silly. I followed this guide
https://supabase.com/docs/guides/cli/getting-started
to update my supabase cli. Did the supabase stop, start etc. I am trying to access a new function called resend()

```
    await supabaseAdmin.auth.resend(...) 
```
But it doesn't exists.
https://supabase.com/docs/reference/javascript/auth-resend
I apologise for the very beginner question.
```
 $ supabase --version               
1.110.1
```

**Comment by GaryAustin1** [comment]
Did you upgrade your supabase-js on your client?

**Comment by RafaelCENG** [comment]
Yes, I updated it. Should I do supabase start/stop again to see the new features?
